tests/accessor/accessor_exceptions_core.cpp

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor exceptions.
+//
+// This test provides verifications that exception really has been thrown for
+// generic accessor, host_accessor and local_accessor with generic types.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+
+#include "accessor_exceptions.hpp"
+
+using namespace accessor_exceptions_test;
+using namespace accessor_tests_common;
+#endif
+
+namespace accessor_exceptions_test_core {
+using namespace sycl_cts;
```

Comment:
Where are we using it?

Reply:
I think nowhere
It has been deleted [here](https://github.com/KhronosGroup/SYCL-CTS/pull/329/commits/42bfbc5615e7ade3ebb872e56b49af872258b109#diff-d5d855436418cccf74b1183a84f04ba9639b9c32425cd2a0204b64b24fc5d348L26)

Reply:
`namespace sycl_cts` are used for getting `queue` objects and util objects like `id` and `range`. In this PR `sycl_cts::` are used in line 45 and line 47 and similar places. 

```  
  auto queue = util::get_cts_object::queue();
  DataT some_data(expected_val);
  auto r = util::get_cts_object::range<Dimension>::get(1, 1, 1);
```

You should either return namespace to code or explicitly add using of namespace in functions invocations.

Reply:
@andreyromanof `auto queue = sycl_cts::util::get_cts_object::queue();` works fine without using `sycl` namespace because of the fact that we use the `sycl` namespace to instance object of the `sycl::queue` class: 
`return sycl::queue(selector.select_device(), asyncHandler, sycl::property_list{});`
This also applies to obtain `sycl::range`

Reply:
@andreyromanof You are right. Your comment has been applied in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/329/commits/efbd40fc846edd77557993ea2ce60d247482bbb5